README.md

```diff
@@ -1,6 +1,6 @@
 # Gozde Arslan . Jarvis Consulting
 
- A hardworking and passionate data engineer. I want to apply my enthusiasm, superb analytical thinking, strong problem- solving skills and previous experience to provide best solutions for my employer.
+After working on several engineering projects that incorporated data-driven software tools, I determined to sharpen my software abilities to put my passion for the software and data industries to effective use. Along with this, I am a recent Seneca College Computer Science graduate. Throughout my studies, I have worked on a variety of research projects in object-oriented programming, database management, and web development. Involved in and handled many team initiatives that aided in the development of various teamwork roles and leadership skills. Working as a data engineer or analyst position piqued my interest. I'm presently employed as a data engineer associate at the Jarvis Consulting Group, where I develop software applications using Linux and Java tools while adhering to the Agile approach.
```

Comment:
After working on several engineering projects that incorporated data-driven software tools, I determined to sharpen my software abilities to put my passion for the software and data industries to practical use. Along with this, I am a recent Seneca College Computer Science graduate. Throughout my studies, I have worked on a variety of research projects in object-oriented programming, database management, and web development. Involved in and handled many team initiatives that aided in the development of various teamwork roles and leadership skills. Working as a software and data engineer or analyst position piqued my interest. I'm presently employed as a data engineer associate at the Jarvis Consulting Group, where I develop software applications using Linux and Java tools while adhering to the Agile approach.
		
minor fix on workding